up42/_version.txt

```diff
@@ -1 +1 @@
-0.8.1b7
\ No newline at end of file
+0.8.2b0
```

Comment:
Unusual versioning. Why do we use the "bX" at the end?

Reply:
Will switch to 0.8.2